roles/splunk/tasks/configure_license.yml

```diff
@@ -5,7 +5,7 @@
 
 - name: Setting licensing mode based on Splunk version number
   set_fact:
-    mode_option: "{% if splunk_version_release | float < 9.0 %}master{% else %}manager{% endif %}_uri"
+    mode_option: "{{ splunk_version_release is version_compare('9.0', '<') | ternary('master_uri', 'manager_uri') }}"
```

Comment:
Good catch. `version_compare` is now `version`. I am using it [here](https://github.com/splunk/ansible-role-for-splunk/blob/master/roles/splunk/tasks/main.yml#L67).

Reply:
I tested, seems to be working fine.

Just a nit-pick, can you just change `version_compare` to `version`? It was renamed in version 2.5 [reference](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_tests.html)